packages/flutter/lib/src/material/text_button.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:ui' show lerpDouble;
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
+import 'package:flutter/src/material/button.dart';
```

Comment:
Ditto of my previous comment.

Reply:
I have done this in the three files, where do i need to put the enum in your opinion?

Reply:
Sorry, the review UI didn't update and I missed your reply. You could take a look at my comment below.